cli/command/container/signals_sigurg.go

```diff
@@ -1,3 +1,5 @@
+// +build linux darwin
```

Comment:
Maybe `signals_unix.go` and make it `!windows`
I saw some issues related to bsd in the go repo about this as well.

It's a pretty good bet no one will want to forward this signal, ever.

Reply:
Was wondering if we should filter by name; does windows also have a "SIGURG" defined?

Reply:
I've pushed an update to use `signals_unix.go` and `!windows`. Hopefully CircleCI's infra is back on line!

It looks like SIGURG isn't defined on Windows, according to https://stackoverflow.com/a/63396672/6774019 . According to the [x/sys/windows docs](https://pkg.go.dev/golang.org/x/sys/windows#Signal) there is only
```golang
const (
	// More invented values for signals
	SIGHUP  = Signal(0x1)
	SIGINT  = Signal(0x2)
	SIGQUIT = Signal(0x3)
	SIGILL  = Signal(0x4)
	SIGTRAP = Signal(0x5)
	SIGABRT = Signal(0x6)
	SIGBUS  = Signal(0x7)
	SIGFPE  = Signal(0x8)
	SIGKILL = Signal(0x9)
	SIGSEGV = Signal(0xb)
	SIGPIPE = Signal(0xd)
	SIGALRM = Signal(0xe)
	SIGTERM = Signal(0xf)
)
```
I guess this is good news 😅